app/utils.py

```diff
@@ -293,8 +294,7 @@ def validate_image(bytestring):
         return False
 
 
-EMAIL_PATTERN = re.compile(r'(?:^|\s)[-a-z0-9_.%$+]+@(?:[-a-z0-9]+\.)+'
-                           '[a-z]{2,63}(?:\s|$)', re.IGNORECASE)
+EMAIL_VALIDATOR = EmailValidator()
```

Comment:
Do we still need this field?

Reply:
We need EMAIL_VALIDATOR, yes. It's what we're using in place of the regex.